libs/blocks/global-navigation/global-navigation.css

```diff
@@ -320,14 +320,23 @@ header.global-navigation {
   flex-shrink: 0;
 }
 
+.feds-breadcrumbs li:last-child:not(:first-child) {
+  padding-right: 12px;
+}
+
+[dir = "rtl"] .feds-breadcrumbs li:last-child:not(:first-child) {
+  padding-right: 0;
+  padding-left: 12px;
+}
+
 /* Hide all breadcrumbs except the first and last two */
 .feds-breadcrumbs li:nth-last-child(n+3):not(:first-child) {
   display: none;
 }
 
 /* If first breadcrumb is not third to last, add ellipsis after it */
 .feds-breadcrumbs li:first-child:not(:nth-last-child(-n+3)):after {
-  content: '/\2003...';
+  content: '/\3000…';
```

Comment:
TIL https://en.wikipedia.org/wiki/Template:Whitespace_(Unicode)

Reply:
I had to research it too. The ideographic space seems to be the only one which is consistent on all browsers, so went for that one